notebooks/data-sources/gcsweb-ci/prow_archive_discovery.ipynb

```diff
@@ -0,0 +1,1159 @@
+{
```

Comment:
<p>not too important, but the notebook cell seems to output the percentage of builds that have the events.json file as 97.93 instead of 96.5?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>